### DISCUSSION TITLE
Change location of zcompdump files

### DISCUSSION THREAD

**Discussion by daankata** [zimfw/zimfw#464] [Q&A] [answered]
I am trying to change the location of zcompdump files. I would like to move them to ~/.cache/zsh so I have less clutter in my home directory. I see there was an [issue](https://github.com/zimfw/zimfw/issues/42) opened on this subject 5 years ago and a commit but it was later changed? What is the best way to do this? I tried with an environment variable zdumpfile in my .zshrc but with no success.

**Comment by ericbn** [comment, marked as answer]
Hi @daankata. That 5 years old code moved to the [completion](https://github.com/zimfw/completion) repository. Check the [settings section under the completion module documentation](https://github.com/zimfw/completion/blob/master/README.md#settings).
